config/locales/en.yml

```diff
@@ -21,31 +21,34 @@ en:
     info:
       nav:
         about: "How this works"
-        privacy: "Privacy Policy"
-        github: "Github"
+        privacy: "Privacy policy"
+        github: "GitHub"
```

Comment:
That was an older PR https://github.com/fugacious/fugacious/pull/118 which was a bit scattered and still had work to be done so I closed it.